tsconfig.json

```diff
@@ -17,5 +17,10 @@
     "isolatedModules": true,
     "types": ["node", "jest", "webdriverio/async"]
   },
-  "exclude": ["src/**/*.test.ts", "src/frontend/generated/*"]
+  "exclude": [
+    "src/**/*.test.ts",
+    "src/frontend/src/test-e2e/*.ts",
+    "src/frontend/generated/*",
+    "src/frontend/custom-sequencer.ts"
```

Comment:
to make more obvious that the custom-sequencer for test, would it be worth renaming it `test-custom-sequencer.ts`?

Reply:
Very good point. Will rename `jest-custom-sequencer.ts`!